eras/shelley/impl/src/Cardano/Ledger/Shelley/Rewards.hs

```diff
@@ -11,41 +11,40 @@ module Cardano.Ledger.Shelley.Rewards
   ( StakeShare (..),
     PoolRewardInfo (..),
     mkApparentPerformance,
-    RewardType (..),
     LeaderOnlyReward (..),
     leaderRewardToGeneral,
-    Reward (..),
     leaderRew,
     memberRew,
     aggregateRewards,
     filterRewards,
     sumRewards,
     rewardOnePoolMember,
     mkPoolRewardInfo,
+    -- deprecated
+    RewardType,
+    Reward,
```

Comment:
This will only introduce unnecessary breakage downstream it is better to just re-export those types from here.